packages/svelte-vscode/package.json

```diff
@@ -637,7 +647,7 @@
                     "group": "4_search"
                 },
                 {
-                    "when": "explorerResourceIsFolder",
+                    "when": "explorerResourceIsFolder && config.svelte.ui.svelteKitFilesContextMenu.enable == always || explorerResourceIsFolder && svelte.uiContext.svelteKitFilesContextMenu.enable",
```

Comment:
Parentheses don't seem to work here. So the `explorerResourceIsFolder` is repeated twice.